sdk/src/settings/mod.rs

```diff
@@ -160,67 +176,126 @@ impl SettingsValidate for Trust {
     }
 }
 
-// TODO: all of these settings aren't implemented
-// Settings for core C2PA-RS functionality
+/// Settings to configure core features.
+#[cfg_attr(feature = "json_schema", derive(schemars::JsonSchema))]
```

Comment:
Shouldn't we keep core features config SDK-internal?

Reply:
I think the core settings in this PR are reasonable to expose to the user.